sundials27.rb

```diff
@@ -0,0 +1,46 @@
+class Sundials27 < Formula
+  desc "Nonlinear and differential/algebraic equations solver"
+  homepage "https://computation.llnl.gov/casc/sundials/main.html"
+  url "https://computation.llnl.gov/projects/sundials/download/sundials-2.7.0.tar.gz"
+  sha256 "d39fcac7175d701398e4eb209f7e92a5b30a78358d4a0c0fcc23db23c11ba104"
+
+  conflicts_with "sundials", :because => "this is an older version"
+
+  option "with-openmp", "Enable OpenMP multithreading"
+
+  depends_on "cmake" => :build
+  depends_on "suite-sparse"
+  depends_on "openblas"
+  depends_on :fortran
+  depends_on :mpi => [:cc, :f77, :recommended]
```

Comment:
This should be `depends_on "open-mpi" => :recommended` now.

Reply:
Core does "with-mpi" for sundials... I guess we should remain compatible.